### PR TITLE
JDO 850: Consolidate download pages

### PR DESCRIPTION
**Pull request by tzaeschke**
This PR consolidates the multipl download pages for 3.2, 3.1, ...  into a single page.

### CODE REVIEW

**Review by clr-apache — APPROVED** [review]
I would remove the 3.1-rc1 (no longer relevant) now that 3.1 is released.
I'd probably also remove 3.0.1 (same reason).
Once the page is reviewed in production the other download pages can be removed.

File: src/main/asciidoc/downloads-all.adoc [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+=== Verifying Releasesanchor:Verifying_Releases[]
+
+anchor:Verifying[]
+
+It is essential that you verify the integrity of the downloaded files
+using the PGP signature.
+
+The PGP signatures can be verified using PGP or GPG. First download the
+link:https://www.apache.org/dist/db/jdo/KEYS[KEYS] as well as the `asc`
```
Comment (clr-apache) [inline src/main/asciidoc/downloads-all.adoc:98]: See above. Use downloads/db/jdo/KEYS here as well.

**Review by clr-apache — COMMENTED** [review]
The six occurrences of dist/db/jdo should be changed to downloads/db/jdo.

File: src/main/asciidoc/downloads-all.adoc (hunk 1 above)
Comment (tzaeschke) [reply src/main/asciidoc/downloads-all.adoc:98]: Fixed

**Pull request merged**